tomviz/DataSource.cxx

```diff
@@ -239,6 +241,29 @@ bool DataSource::serialize(pugi::xml_node& ns) const
   tomviz::serialize(this->colorMap(), node);
 
   node = ns.append_child("OpacityMap");
+
+  // Update the proxy values from the VTK object.
+  // TODO - find a nicer way to ensure these values are synchronized.
```

Comment:
Yes. There isn't really a simple way to do this in ParaView, e.g., a single function call, but we may want to update the property in a slot that is invoked every time a signal is fired when the opacity map is changed.